### PR TITLE
created a header and navBar, connected it to the backend

### PR DESCRIPTION
**Pull request by Hacene010**

### CODE REVIEW

**Review by GeorgesFORTRY — DISMISSED** [review]
all is ok !

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,3 +1,89 @@
...
+import { useEffect, useState } from 'react';
+import SHeader from './Style';
+
 export default function Header() {
-  return <div>Header</div>;
+  const [translations, setTranslations] = useState({});
+  const [images, setImages] = useState({});
+
+  useEffect(() => {
+    axios.get('http://localhost:8080/texts').then(({ data }) => {
```
Comment (LoicBrassart) [inline src/components/Header/Header.jsx]: You shouldn't use a hardcoded "http://localhost:8080" value: once in production, it won't work anymore.
Create a `.env` in your frontend folder and move this value there, same as the backend!

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -1,3 +1,89 @@
...
+      setTranslations(data);
+    });
+    axios.get('http://localhost:8080/images').then(({ data }) => {
```
Comment (LoicBrassart) [inline src/components/Header/Header.jsx]: Same here

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One more thing to fix!

**Review by GeorgesFORTRY — APPROVED** [review]
it's good 😅

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**